addons/l10n_bh/data/tax_report_simplified.xml

```diff
@@ -0,0 +1,285 @@
+                            </record>
+                        </field>
+                    </record>
+                </field>
+            </record>
+            <record id="l10n_bh_tax_report_simplified_3" model="account.report.line">
+                <field name="name">3. Other and exempt sales</field>
+                <field name="code">bh_simp_3</field>
+                 <field name="foldable" eval="True"/>
+                <field name="expression_ids">
+                    <record id="l10n_bh_tax_report_simplified_3_base" model="account.report.expression">
+                        <field name="label">base</field>
+                        <field name="engine">aggregation</field>
+                        <field name="formula">bh_3_1.base + bh_3_2.base</field>
+                    </record>
+                </field>
+                <field name="children_ids">
+                    <record id="l10n_bh_tax_report_simplified_3_1" model="account.report.line">
+                        <field name="name">Sales subject to domestic reverse charge mechanism</field>
+                        <field name="code">bh_3_1</field>
+                        <field name="expression_ids">
+                            <record id="l10n_bh_tax_report_simplified_3_1_base" model="account.report.expression">
+                                <field name="label">base</field>
+                                <field name="engine">tax_tags</field>
+                                <field name="formula">3 B</field>
+                            </record>
+                        </field>
+                    </record>
+                    <record id="l10n_bh_tax_report_simplified_3_2" model="account.report.line">
+                        <field name="name">Exempt</field>
+                        <field name="code">bh_3_2</field>
+                        <field name="expression_ids">
+                            <record id="l10n_bh_tax_report_simplified_3_2_base" model="account.report.expression">
+                                <field name="label">base</field>
+                                <field name="engine">tax_tags</field>
+                                <field name="formula">6 B</field>
+                            </record>
+                        </field>
+                    </record>
+                </field>
+            </record>
+            <record id="l10n_bh_tax_report_simplified_7" model="account.report.line">
+                <field name="name">7.Total sales</field>
+                <field name="code">bh_simp_7</field>
+                <field name="expression_ids">
+                    <record id="l10n_bh_tax_report_simplified_7_base" model="account.report.expression">
+                        <field name="label">base</field>
+                        <field name="engine">aggregation</field>
+                        <field name="formula">bh_simp_1a.base + bh_simp_2.base + bh_simp_3.base </field>
```

Comment:
Maybe it's normal with the use of the simplified, but the tag 2B does not appear here (and so you don't have the same amounts in simplified and full, which seems weird)
I would expect it to be included in the category 2 of the simplified too